spec/features/users/sign_in_spec.rb

```diff
@@ -116,30 +116,16 @@
 
     context 'javascript enabled', js: true do
       it 'pops up session timeout warning' do
-        # session of length 0.2.minutes == 12 sec
-        # we need session timeout long enough to allow session to live
-        # between requests (esp at travis), but short enough so that
-        # using_wait_time does not expire during find_link().
-        session_ttl = 0.2.minutes # 12 sec
-        page_wait_time = 15 # seconds, must be >= session_ttl
+        session_ttl = 0.01.minutes # 0.6 sec
+        page_wait_time = 1 # seconds, must be >= session_ttl
 
         allow(Devise).to receive(:timeout_in).and_return(session_ttl)
 
-        user = sign_in_and_2fa_user
-        click_link(t('links.sign_out'))
+        visit root_path
 
         Capybara.using_wait_time(page_wait_time) do
-          find_link(t('forms.buttons.continue')).trigger('click')
+          expect(page).to have_link(t('forms.buttons.continue'))
```

Comment:
this test existed before the modal was added, and was adapted to expect the modal. The original description was a little different than the test at line 91. This test is "sign in, then out, then in again". The other test is "sign in".

I'm ok to nuke it, but let's do that in a different PR.